librosa/filters.py

```diff
@@ -872,3 +873,89 @@ def get_window(window, Nx, fftbins=True):
                              '{:d} != {:d}'.format(len(window), Nx))
     else:
         raise ParameterError('Invalid window specification: {}'.format(window))
+
+
+@cache(level=10)
+def multirate_fb(center_freqs, sample_rates, Q=25, passband_ripple=1, stopband_attenuation=50, ftype='ellip'):
+    r'''Construct a multirate filterbank.
+
+     Uses `scipy.signal.iirdesign` to design the filters.
+     the filter bank described in [1]_.
+
+    .. [1] Müller, Meinard.
+           "Information Retrieval for Music and Motion."
+           Springer Verlag. 2007.
+
+
+    Parameters
+    ----------
+    center_freqs : np.ndarray
+        Filters' center frequencies.
+    Q : float
+        Q factor (influences the filter bandwith).
+    passband_ripple : float
+        The maximum loss in the passband (dB) (cf. `scipy.signal.iirdesign`).
+    stopband_attenuation : float
+        The minimum attenuation in the stopband (dB) (cf. `scipy.signal.iirdesign`).
+    ftype : str
+        The type of IIR filter to design (cf. `scipy.signal.iirdesign`).
+
+    Returns
+    -------
+    filterbank
+
+    Notes
+    -----
+    This function caches at level 10.
+
+    See Also
+    --------
+    librosa.core.cqt
+
+
+    Examples
+    --------
+    >>> import matplotlib.pyplot as plt
+    >>> import numpy as np
+    >>> import scipy.signal
+    >>> pitch_filterbank, sample_rates, _, _ = librosa.filters.multirate_pitch_fb()
+    >>> plt.figure(figsize=(10, 6))
+    >>> for cur_sr, cur_filter in zip(sample_rates, pitch_filterbank):
+    ...    w, h = scipy.signal.freqz(cur_filter[0], cur_filter[1], worN=2000)
+    ...    plt.plot((cur_sr / (2 * np.pi)) * w, 20 * np.log10(abs(h)))
+    >>> plt.semilogx()
+    >>> plt.xlim([20, 10e3])
+    >>> plt.ylim([-60, 3])
+    >>> plt.title('Magnitude Responses of the Pitch Filterbank')
+    >>> plt.xlabel('Log-Frequency (Hz)')
+    >>> plt.ylabel('Magnitude (dB)')
+    >>> plt.tight_layout()
+    '''
+
+    nyquist = sample_rates / 2
+    filter_bandwidths = center_freqs / Q
+
+    filterbank = []
+
+    for cur_center_freq, cur_nyquist, cur_bw in zip(center_freqs, nyquist, filter_bandwidths):
+        passband_freqs = [cur_center_freq - cur_bw / 2, cur_center_freq + cur_bw / 2] / cur_nyquist
+        stopband_freqs = [cur_center_freq - cur_bw, cur_center_freq + cur_bw] / cur_nyquist
+
+        cur_filter = scipy.signal.iirdesign(passband_freqs, stopband_freqs,
+                                            passband_ripple, stopband_attenuation,
+                                            analog=False, ftype=ftype, output='ba')
+
+        filterbank.append(cur_filter)
+
+    return filterbank
+
+
+def multirate_fb_ct(center_freqs=midi_to_hz(np.arange(21, 121), A440=440),
```

Comment:
I think I don't understand your question.

Is it whether `multirate_fb_ct` should exist, or where it should be placed?  I don't think I can answer that prior to CR, since I don't really know how things are structured or what the different pieces do.